joint_pd_controller:
    type: fr3_ros/JointPDController
    arm_id: $(arg arm_id)
    joint_names:
        - $(arg arm_id)_joint1
        - $(arg arm_id)_joint2
        - $(arg arm_id)_joint3
        - $(arg arm_id)_joint4
        - $(arg arm_id)_joint5
        - $(arg arm_id)_joint6
        - $(arg arm_id)_joint7
    k_gains:
        - 700.0
        - 400.0
        - 500.0
        - 600.0
        - 5000.0
        - 3000.0
        - 5000.0
    d_gains:
        - 60.0
        - 30.0
        - 40.0
        - 20.0
        - 500.0
        - 400.0
        - 1000.0
    # k_gains:
    #     - 600.0
    #     - 600.0
    #     - 600.0
    #     - 600.0
    #     - 250.0
    #     - 150.0
    #     - 50.0
    # d_gains:
    #     - 50.0
    #     - 50.0
    #     - 50.0
    #     - 20.0
    #     - 20.0
    #     - 20.0
    #     - 10.0
    radius: 0.1
    acceleration_time: 2.0
    vel_max: 0.15
    publish_rate: 10.0
    coriolis_factor: 1.0
    p_gain: 1000.0
    d_gain: 10.0
    dq_gain: 1.0